PowershellTools.Common/Constants.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace PowerShellTools.Common
+{
+    internal static class Constants
+    {
+        public const string ProcessManagerHostUri = "net.pipe://localhost/";
+
+        public const string ProcessManagerHostRelativeUri = "NamedPipePowershellProcess";
+
+        public const string ProcessManagerDataNamespace = "https://github.com/Microsoft/poshtools";

```

Comment:
We shouldn't have this a github-repo linked URI for the namespace.  [Thinking out loud, do we even need a namespace for this?]